README.md

```diff
@@ -208,3 +208,317 @@ Explore victoriaMetrics go client <https://github.com/VictoriaMetrics/metrics>
+            }
+        ],
+        [
+            {
+                "data": 3
+            },
+            {
+                "data": "CANADA"
+            },
+            {
+                "data": 1
+            },
+            {
+                "data": "eas hang ironic, silent packages. slyly regular packages are furiously over the tithes. fluffily bold"
+            }
+        ],
+        [
+            {
+                "data": 4
+            },
+            {
+                "data": "EGYPT"
+            },
+            {
+                "data": 4
+            },
+            {
+                "data": "y above the carefully unusual theodolites. final dugouts are quickly across the furiously regular d"
+            }
+        ],
+        [
+            {
+                "data": 5
+            },
+            {
+                "data": "ETHIOPIA"
+            },
+            {
+                "data": 0
+            },
+            {
+                "data": "ven packages wake quickly. regu"
+            }
+        ],
+        [
+            {
+                "data": 6
+            },
+            {
+                "data": "FRANCE"
+            },
+            {
+                "data": 3
+            },
+            {
+                "data": "refully final requests. regular, ironi"
+            }
+        ],
+        [
+            {
+                "data": 7
+            },
+            {
+                "data": "GERMANY"
+            },
+            {
+                "data": 3
+            },
+            {
+                "data": "l platelets. regular accounts x-ray: unusual, regular acco"
+            }
+        ],
+        [
+            {
+                "data": 8
+            },
+            {
+                "data": "INDIA"
+            },
+            {
+                "data": 2
+            },
+            {
+                "data": "ss excuses cajole slyly across the packages. deposits print aroun"
+            }
+        ],
+        [
+            {
+                "data": 9
+            },
+            {
+                "data": "INDONESIA"
+            },
+            {
+                "data": 2
+            },
+            {
+                "data": " slyly express asymptotes. regular deposits haggle slyly. carefully ironic hockey players sleep blithely. carefull"
+            }
+        ]
+    ]
+}
+```
+
+**Reponse(error example):**
+
+```json
+{
+    "status": "Error",
+    "error": {
+        "message": "Unable to query trino: trino: query failed (200 OK): \"USER_ERROR: line 1:31: mismatched input '10'. Expecting: ',', '.', 'AS', 'CROSS', 'EXCEPT', 'FETCH', 'FOR', 'FULL', 'GROUP', 'HAVING', 'INNER', 'INTERSECT', 'JOIN', 'LEFT', 'LIMIT', 'MATCH_RECOGNIZE', 'NATURAL', 'OFFSET', 'ORDER', 'RIGHT', 'TABLESAMPLE', 'UNION', 'WHERE', 'WINDOW', <EOF>, <identifier>\"",
+        "errorCode": 500,
+        "errorName": "",
+        "errorType": ""
+    }
+}
+```
+## Project Structure
+
+```plaintext
+trino-api/
```

Comment:
Merge it with main repo

lets use `trino-rest` for the name


u can choose suitable names when merging

instead of `app` use `trino-rest` wherever required, for instance when merging config files, etc.

Reply:
same for Dockerfiles 

Although we will not be needing the dockerfiles once the module is merged, for now you can rename them to Dockerfile.trino_rest or something.

Reply:
i think i'll need help in merging to main module, as need to know what things need to omit and what to keep or is there any scope of code changes or refactoring

Reply:
`trino_rest/build` goes into `/build` 

`/trino_rest/cmd/api/main.go` goes into `/cmd/trino_rest/main.go`

merge the config files

`trino_rest/internal` goes into `/internal/trino_rest`

`trino_rest/internal/boot` can be omitted

`trino_rest/internal/config` merge with `/internal/config`

`/trino_rest/pkg` can either be omitted and necessary changes done to the code to use libraries in `/pkg`
OR they can be moved to `internal/trino_rest/pkg`


go.mod and go.sum will be merged

Reply:
i wanted to discuss that how can we make sure that the integration with main module is working